packages/driver/cypress/e2e/commands/prompt/prompt.cy.ts

```diff
@@ -3,22 +3,31 @@ describe('src/cy/commands/prompt', () => {
     Cypress.testingType = 'e2e'
   })
 
-  it('executes the prompt command', () => {
-    // TODO: (cy.prompt) We will look into supporting other browsers
-    // as this is rolled out. We will add error messages for other browsers
-    // and add tests if necessary
-    if (Cypress.isBrowser('webkit') || Cypress.isBrowser('firefox')) {
-      return
+  // TODO: (cy.prompt) We will look into supporting other browsers
+  // as this is rolled out. We will add error messages for other browsers
+  // and add tests if necessary
+  if (!Cypress.isBrowser('webkit') && !Cypress.isBrowser('firefox')) {
+    const contributorPr = Cypress.env('CI') && !Cypress.env('RECORD_KEY') && Cypress.config('isTextTerminal')
```

Comment:
does this need to be in `Cypress.env('CI')` or can we just detect the `process.env` variable here since we shim it with the webpack preprocessor?